actioncable/Rakefile

```diff
@@ -11,3 +13,40 @@ Rake::TestTask.new do |t|
   t.verbose = true
   t.ruby_opts = ["--dev"] if defined?(JRUBY_VERSION)
 end
+
+namespace :assets do
+  task :precompile do
+    ## Check for coffee executable
+    require 'mkmf'
+    unless find_executable('coffee')
+      raise "In order to compile Action Cable's assets, you must have the "\
+        " coffee-script npm package installed."
+    end
+
+    Dir.chdir(ASSETS_ROOT) do
+      puts 'Compiling Action Cable front-end Coffee Script to JS...'
+      `coffee --output action_cable_js --compile action_cable`
+
+      ## Add requires for consumer.js
+      f = File.open("#{ASSETS_ROOT}/action_cable_js/consumer.js", "r+")
+      lines = f.readlines
+      f.close
+
+      lines = [
+        "//= require action_cable/connection\n",
+        "//= require action_cable/connection_monitor\n",
+        "//= require action_cable/subscriptions\n",
+        "//= require action_cable/subscription\n"
```

Comment:
If we compile via sprockets, as @rafaelfranca suggested, then we should be able to skip these complications, and just produce a single combined JS file (plus a require for the 'internal' thing, mentioned below).